packages/flutter_test/lib/src/binding.dart

```diff
@@ -1116,23 +1116,23 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
   void _verifyInvariants() {
     super._verifyInvariants();
 
-    assert(() {
-      if (   _currentFakeAsync.periodicTimerCount == 0
-          && _currentFakeAsync.nonPeriodicTimerCount == 0) {
-        return true;
-      }
+    bool timersPending = false;
+    if (   _currentFakeAsync.periodicTimerCount != 0
+        || _currentFakeAsync.nonPeriodicTimerCount != 0) {
+
+        debugPrint('Pending timers:');
+        for (final FakeTimer timer in _currentFakeAsync.pendingTimers) {
+          debugPrint(
+            'Timer (duration: ${timer.duration}, '
+            'periodic: ${timer.isPeriodic}), created:');
+          debugPrintStack(stackTrace: timer.creationStackTrace);
+          debugPrint('');
+        }
 
-      debugPrint('Pending timers:');
-      for (final FakeTimer timer in _currentFakeAsync.pendingTimers) {
-        debugPrint(
-          'Timer (duration: ${timer.duration}, '
-          'periodic: ${timer.isPeriodic}), created:');
-        debugPrintStack(stackTrace: timer.creationStackTrace);
-        debugPrint('');
-      }
-      return false;
-    }(), 'A Timer is still pending even after the widget tree was disposed.');
+        timersPending = true;
+    }
 
+    assert(!timersPending ? true : throw 'A Timer is still pending even after the widget tree was disposed.');
```

Comment:
nit: `assert` takes two parameters, `assert(condition, message)`

Reply:
I fixed the assert.